tamr_toolbox/low_latency/llm.py

```diff
@@ -0,0 +1,211 @@
+import json
+import logging
+import requests
+import time
+from collections import defaultdict
+from typing import Dict, List, Optional, Union
+
+from tamr_unify_client import Client
+from tamr_unify_client.operation import Operation
+from tamr_toolbox.utils.operation import from_resource_id
+
+from tamr_toolbox.models.data_type import JsonDict
+
+LOGGER = logging.getLogger(__name__)
+
+
+def update_llm_data(
+    client: Client,
+    *,
+    project_name: str,
```

Comment:
Good catch, LLM endpoints need the `name`, not the `displayName`. Will work on addressing this correctly.

Reply:
There is no versioned endpoint to get the `name` of a project if it has been renamed.  I just learned that recent versions of Tamr have a new set of LLM endpoints (containing `namespace` in the path), which take the project ID instead.  This is what we should use for this functionality.  It may make sense to do a version-dependent implementation that takes a `Project` as a parameter, and then extracts the `name` attribute for the older versions (which will fail if the project has been renamed) and extracts the `id` attribute for the newer versions (which will be robust).

Reply:
Since there is a workaround to get the original name, I will try to do that for the moment, and we can discuss later adding better handling for the latest LLM versions.

Reply:
Taking a `Project` as the parameter instead of a `str` project name will make this easier to make backwards-compatible.